examples/low-level/old/ulib/IntLib.fst

```diff
@@ -17,7 +17,7 @@ module IntLib
 
 open Axioms
 
-(** Mathematical useful fonctions **)
```

Comment:
I think this change loses a bit of the original sense. Maybe "Useful functions for mathematical integers" ?

Reply:
Seems like bikeshedding...